config/routes.rb

```diff
@@ -1,4 +1,8 @@
 Rails.application.routes.draw do
   devise_for :users
-  # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+
+  resources :tweets
```

Comment:
If you do this, you will generate URI's for all 7 routes. However, your controller only defines 1 of the methods. 

What will happen is users who type in a URI that you haven't specified in your controller yet, will receive a nasty error message. 

I suggest putting an `only: [:index]` here. This way, when you add more functionality, you can add more methods into the array.